hz.ps1

```diff
@@ -2562,6 +2590,30 @@ function hz-getfwks-json {
     ConvertTo-Json -InputObject $frameworks -Compress
 }
 
+## Copies files from given path to project folder with respect to source hierarhcy.
+function hz-copy-source (){
```

Comment:
not sure I understand the nature of the function from its name. is the purpose to copy "some files" from "somewhere" to "somewhere else"? then I'd rename the command as `copy-files` and its corresponding option as `copy-files-source` to be super explicit. we might have something else having a "source" in the future.